python/packages/ai/teams/ai/models/openai_model.py

```diff
@@ -271,7 +271,34 @@ async def complete_prompt(
                     if delta.content:
                         message_content += delta.content
 
-                    # TODO: Handle tool calls
+                    # Handle tool calls during streaming
+                    if is_tools_aug and delta.tool_calls:
+                        if not hasattr(message, "action_calls") or message.action_calls is None:
+                            message.action_calls = []
+
+                        for curr_tool_call in delta.tool_calls:
+                            index = curr_tool_call.index
+
+                            # Ensure the action_calls array is long enough
+                            while index >= len(message.action_calls):
+                                message.action_calls.append(
+                                ActionCall(
+                                        id="",
+                                        function=ActionFunction(name="", arguments=""),
+                                        type="function",  # /python/packages/ai/teams/ai/prompts/message.py#L123
```

Comment:
remove commented line

Reply:
was just leaving that as a reference for why I selected type="function". 

It is currently the only supported type

@dataclass_json
@dataclass
class ActionCall(DataClassJsonMixin):
    """
    Action called by the model.
    """

    id: str
    """The ID of the action call."""

    function: ActionFunction
    """The function that the model called."""

    type: Literal["function"]
    """The type of the action. Currently, only `function` is supported."""

Reply:
or I many be misunderstanding the comment